src/app/app.currentWeather.js

```diff
@@ -0,0 +1 @@
+export default {};
```

Comment:
`app.` prefix is not necessary, that prefix format is used only for component related files.
a better name could be `currentWeather.value.js` or other options is moved all values for a folder called values and the name can be just `currentWeather.js`.
We have to pursuit intuitive names and paths in order to find everything asap :)